### ISSUE TITLE
how is this design?

### ISSUE THREAD

**Issue by uqiu** [happiness86/references-plus#3]
![图片](https://github.com/happiness86/references-plus/assets/25265788/43c40632-2939-4ae7-9c0f-41fc5c24cac9)

easier to distinguish each group of results.

idea from search page

![图片](https://github.com/happiness86/references-plus/assets/25265788/fd6a256f-48d6-45ba-8092-c6a8ce452286)

**Comment by happiness86** [comment]
Adding highlighting at the root node is really clearer。

**Comment by happiness86** [comment]
next version!

**Comment by uqiu** [comment]
I've also done some research on how to achieve this effect.
It seems not easy to implement, as this VSCODE component doesn't seem to support much customization with CSS.

btw: whit this [fix](https://github.com/happiness86/references-plus/issues/4). now references-plus works like a charm. 👍

**Comment by happiness86** [comment]
Yes, tree view component supports only limited customization.But I found that the badge attribute might fit the requirement.
